docs/fundamentals/toc.yml

```diff
@@ -2099,6 +2099,47 @@ items:
     href: ../core/extensions/generic-host.md
   - name: HTTP with .NET
     href: ../core/extensions/http-client.md
+  - name: Globalization and localization
```

Comment:
I'm not sure this belongs under "Runtime libraries"? Consider moving it up a level to be a peer of "Runtime libraries".

Reply:
But that means we would have content specific to extensions not under extensions, and I really want to keep that together. What do you suggest?

Reply:
How about we split the localization article up into two - one that's specific to extensions and one that's more generally about localization. Are the extensions the only way to obtain localized strings? Keep the extensions-localization article in the extensions subfolder and "runtime libraries" toc, add a glob/loc node as a peer to "runtime libraries", and be sure to link to the extensions-localization article from the general localization article.